windows/start.sh

```diff
@@ -84,6 +84,8 @@ docker () {
 }
 export -f docker
 
+set +e
```

Comment:
If we're going to go with this approach, would be good to add a comment explaining the motivation, since generally I find `set +e` pretty incongruous in scripts.